rs/nns/governance/src/governance/voting_power_snapshots_tests.rs

```diff
@@ -39,60 +39,72 @@ fn test_record_voting_power_snapshot() {
 
     // After making a snapshot, the latest snapshot timestamp should be the timestamp of the
     // snapshot.
-    snapshots.record_voting_power_snapshot(1, voting_power_snapshot(vec![9], 10));
+    snapshots.record_voting_power_snapshot(1, voting_power_snapshot(vec![90], 100));
     assert_eq!(snapshots.latest_snapshot_timestamp_seconds(), Some(1));
-    // We should disable early adoption if the deciding voting power is more than 2 times the
+    // We should return previous ballots if the deciding voting power is more than 2 times the
     // minimum voting power in the first snapshot.
```

Comment:
> 2

Is this right? I'm comparing 2 * 100 vs. 149 and 151, right? 149 and 151 seem to be in the same regime wrt 2 * 100...